### PR TITLE
v10.3 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by agnivade — APPROVED** [review]

File: source/about/mattermost-v10-changelog.md [hunk 1]
```diff
@@ -7,6 +7,78 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
+ - Updated product string for clarity.
+ - Removed most places where deprecated translation code is used in the web app.
+ - Removed some duplicate CSS from the web app bundle.
+
+#### Administration
+ - A 200 response is now returned for HEAD requests to a sub-path rather than responding with a 302. This fixes mobile devices trying to connect to a server hosted on a sub-path.
+ - Added the ``fetchMissingUsers`` option to ``PostUtils.messageHtmlToComponent`` for use by plugins.
+ - Exporting bot users now functions in a more proper way.
+ - The message export code now lives in the source available enterprise folder.
+ - Added a warning text to mmctl when user specifies per-page parameter larger than maximum.
```
Comment (cwarnermm) [inline source/about/mattermost-v10-changelog.md:45]:
```suggestion
 - Added a warning to mmctl for cases where a user specifies a per-page parameter that's larger than the maximum value supported.
```

File: source/about/mattermost-v10-changelog.md [hunk 2]
```diff
@@ -7,6 +7,78 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
+ - Updated ``API POST /api/v4/user`` to use a proper error message.
+
+#### Performance
+ - Added Desktop App performance metrics.
+
+### Bug Fixes
+ - Fixed an issue with post drafts being unnecessarily saved when changing channels.
+ - Fixed an issue where the Web App would feel slower to load in the Desktop App.
```
Comment (cwarnermm) [inline source/about/mattermost-v10-changelog.md:53]: Slower to load "in" or "than" the Desktop App? What's intended?

File: source/about/mattermost-v10-changelog.md [hunk 3]
```diff
@@ -7,6 +7,78 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
+ - Fixed an issue with a link in the Compliance Monitoring page banner in the System Console.
+ - Fixed an issue that no longer allowed managing user tokens via the System Console.
+ - Fixed a SVG image rendering issue by setting conditional width and height attributes in ``ImagePreview`` and ``SizeAwareImage`` components.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``ServiceSettings `` in ``config.json``:
+    - Added ``ScheduledPosts`` to add a feature to schedule a post for the future.
```
Comment (cwarnermm) [inline source/about/mattermost-v10-changelog.md:65]:
```suggestion
    - Added ``ScheduledPosts`` to add a feature to schedule and send message in the future.
```

**Review by cwarnermm — APPROVED** [review]
Provided editorial feedback inline. Approving.

**Review by streamer45 — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

File: source/about/mattermost-v10-changelog.md [hunk 4]
```diff
@@ -7,6 +7,78 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
 ```
 
+(release-v10.3-feature-release)=
+## Release v10.3 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release Day: December 16, 2024**
+
+### Important Upgrade Notes
+
+ - As part of our continuous efforts to enhance your user experience, we have phased out the Classic Mobile App. Please download the new v2 Mobile App from the [Apple App Store](https://apps.apple.com/us/app/mattermost/id1257222717) or [Google Play Store](https://play.google.com/store/apps/details?id=com.mattermost.rn) to continue enjoying our services without interruption. See more details in the [classic mobile app deprecation](https://forum.mattermost.com/t/classic-mobile-app-deprecation/18703) Mattermost forum post.
```
Comment (lieut-data) [inline source/about/mattermost-v10-changelog.md:17]: nit: The introductory phrasing has a "corporate vibe" to it. Would we consider just omitting it, stating the changes, and linking to the forum post?

**Review by lieut-data — APPROVED** [review]
Nothing blocking, just added a few observations for your consideration @amyblais! 0/5, and will defer to our existing practices here :)

File: source/about/mattermost-v10-changelog.md [hunk 5]
```diff
@@ -7,6 +7,78 @@ Support for Mattermost Server v9.5 [Extended Support Release](https://docs.matte
...
+ - Fixed an issue that no longer allowed managing user tokens via the System Console.
+ - Fixed a SVG image rendering issue by setting conditional width and height attributes in ``ImagePreview`` and ``SizeAwareImage`` components.
+ - Fixed an issue with the web app status not being updated correctly for the current user.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``ServiceSettings `` in ``config.json``:
+    - Added ``ScheduledPosts`` to add a feature to schedule and send message in the future.
```
Comment (marianunez) [inline source/about/mattermost-v10-changelog.md:63]:
```suggestion
    - Added ``ScheduledPosts`` to enable the feature to schedule and send message in the future.
```

**Review by marianunez — APPROVED** [review]

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Pull request merged**